src/backend/BSConfig.pm.template

```diff
@@ -155,8 +154,11 @@ our $relsync_pool = {
 #our $sign = '/usr/bin/sign';
 #Extend sign call with project name as argument "--project $NAME"
 #our $sign_project = 1;
+#Create a key by default for new projects, if top level project don't have one
+#our $forceprojectkeys = 1;
 #Global sign key 
 #our $keyfile = '/srv/obs/openSUSE-Build-Service.asc';
+#our $gpg_standard_key = "/etc/obs-default-gpg.asc";
```

Comment:
yep, but IMHO we can drop it in this template file. (The file is for new installations only and I think it only confuses people there).